<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumEditorCustomGaussian</class>
 <widget class="QWidget" name="SpectrumEditorCustomGaussian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelFunctionExp">
     <property name="text">
      <string>f(x)=Aexp( - (x-μ)^2 / (2σ^2)  )</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>A:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineAmplitude"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>μ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineMu"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>σ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSigma"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
